### PR TITLE
PODS-9717: Twirl migration for Address,Amend,Company & Deregister folders

### PR DESCRIPTION
**Pull request by dynamicwave4**

### CODE REVIEW

File: app/views/address/UseAddressForContactView.scala.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                classes = "govuk-radios--inline"
+            )
+        )
+
+        @govukButton(
+            Button(
+                content = Text(messages("site.save_and_continue")),
+                inputType = Some("submit")
+            )
+        )
```
Comment (PavelVjalicin) [inline app/views/address/UseAddressForContactView.scala.html:73]: Does not have as form submit

File: app/views/address/UseAddressForContactView.scala.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[Boolean], radios: Seq[RadioItem], entityType: String, entityName: String, address: Seq[String])(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+@layout(messages("useAddressForContact.title", messages(entityType))) {
```
Comment (PavelVjalicin) [inline app/views/address/UseAddressForContactView.scala.html:28]: No form has errors parameter

File: app/views/company/IsCompanyRegisteredInUkView.scala.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[Boolean], radios: Seq[RadioItem])(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+@layout(messages("isCompanyRegisteredInUk.title")) {
```
Comment (PavelVjalicin) [inline app/views/company/IsCompanyRegisteredInUkView.scala.html:28]: no form has errors parameter

File: app/views/company/IsCompanyRegisteredInUkView.scala.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                errorMessage = field.error.map(e => ErrorMessage.errorMessageWithDefaultStringsTranslated(content = Text(messages(e.message, e.args:_*))))
+            )
+        )
+
+        @govukButton(
+            Button(
+                content = Text(messages("site.save_and_continue")),
+                inputType = Some("submit")
+            )
+        )
```
Comment (PavelVjalicin) [inline app/views/company/IsCompanyRegisteredInUkView.scala.html:62]: no form as submit

File: app/views/deregister/company/DeregistrationDateView.scala.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        govukInput: GovukInput,
+        govukDateInput: GovukDateInput,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, pspName: String, form: Form[java.time.LocalDate], returnUrl: String, applicationDate: String, viewmodel: uk.gov.hmrc.viewmodels.DateInput.ViewModel)(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+
+@layout(messages("deregistrationDate.company.title")) {
```
Comment (PavelVjalicin) [inline app/views/deregister/company/DeregistrationDateView.scala.html:30]: no form has errors

File: app/views/deregister/individual/DeregistrationDateView.scala.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukDateInput: GovukDateInput,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[java.time.LocalDate], returnUrl: String, applicationDate: String, viewmodel: uk.gov.hmrc.viewmodels.DateInput.ViewModel)(implicit request: Request[_], messages: Messages)
+
+
+@layout(messages("deregistrationDate.individual.title")) {
```
Comment (PavelVjalicin) [inline app/views/deregister/individual/DeregistrationDateView.scala.html:29]: no form has errors

File: app/views/individual/EmailView.scala.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[String])(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+
+@layout(messages("individual.email.title")) {
```
Comment (PavelVjalicin) [inline app/views/individual/EmailView.scala.html:29]: no form has errors

File: app/views/individual/IsThisYouView.scala.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+        govukRadios: GovukRadios,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[Boolean], radios: Seq[RadioItem], name: String, address: Seq[String])(implicit request: Request[_], messages: Messages)
+
+
+@field = @{form("value")}
+
+@layout(messages("isThisYou.title")) {
```
Comment (PavelVjalicin) [inline app/views/individual/IsThisYouView.scala.html:30]: no form has errors

File: app/views/individual/PhoneView.scala.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, form: Form[String])(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+
+@layout(messages("individual.phone.title")) {
```
Comment (PavelVjalicin) [inline app/views/individual/PhoneView.scala.html:29]: no form has errors

File: app/views/individual/PostcodeView.scala.html [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+        govukErrorSummary: GovukErrorSummary,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(submitCall: Call, enterManuallyUrl: String, form: Form[String])(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+
+@layout(messages("individual.postcode.title")) {
```
Comment (PavelVjalicin) [inline app/views/individual/PostcodeView.scala.html:29]: no form has errors

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]

File: app/views/address/UseAddressForContactView.scala.html (hunk 1 above)
Comment (dynamicwave4) [reply app/views/address/UseAddressForContactView.scala.html]: Done

File: app/views/address/UseAddressForContactView.scala.html (hunk 2 above)
Comment (dynamicwave4) [reply app/views/address/UseAddressForContactView.scala.html]: done

File: app/views/company/IsCompanyRegisteredInUkView.scala.html (hunk 3 above)
Comment (dynamicwave4) [reply app/views/company/IsCompanyRegisteredInUkView.scala.html]: done

File: app/views/deregister/company/DeregistrationDateView.scala.html (hunk 5 above)
Comment (dynamicwave4) [reply app/views/deregister/company/DeregistrationDateView.scala.html]: Done

File: app/views/individual/EmailView.scala.html (hunk 7 above)
Comment (dynamicwave4) [reply app/views/individual/EmailView.scala.html]: Done

File: app/views/individual/IsThisYouView.scala.html (hunk 8 above)
Comment (dynamicwave4) [reply app/views/individual/IsThisYouView.scala.html]: Done

File: app/views/individual/PostcodeView.scala.html (hunk 10 above)
Comment (dynamicwave4) [reply app/views/individual/PostcodeView.scala.html]: Done

**Review by PavelVjalicin — APPROVED** [review]

**Pull request merged**